assets/js/base/context/cart-checkout/payment-methods/reducer.ts

```diff
@@ -24,7 +24,7 @@ const reducer = (
 	state = DEFAULT_PAYMENT_DATA_CONTEXT_STATE,
 	{
 		type,
-		paymentMethodData = {},
+		paymentMethodData = undefined,
```

Comment:
What is the benefit of setting this parameter default to `undefined` instead of not setting it?
```suggestion
		paymentMethodData,
```

Reply:
doh ya :) it'll be undefined anyways if the property isn't present.